index.html

```diff
@@ -0,0 +1,336 @@
+            tristique diam tellus id tellus. In consectetur aliquet
+            tristique. Curabitur imperdiet leo in nisl consectetur mattis.
+            Quisque bibendum, mauris vitae suscipit ornare; est nibh
+            elementum mauris, eu malesuada nunc magna at justo. Proin
+            blandit id tellus non pellentesque. Mauris […]
+          </p>
+          <button class="btn btn-outline-secondary">READ MORE</button>
+        </div>
+      </div>
+      <div class="carousel-item">
+        <img class="d-block w-100" src="/starter-code/img/slider-2.jpg" alt="Fourth slide" />
+        <div class="carousel-caption d-none d-md-block">
+          <h5>FUSCE SCELERISQUE NIBH AC NISL LUCTUS INTERDUM.</h5>
+          <p>
+            Nam luctus velit ultricies vestibulum eleifend. Proin imperdiet
+            varius rutrum. Curabitur placerat lacus eu ligula pellentesque
+            euismod? Aliquam volutpat augue id nibh auctor, at varius dui
+            semper? Suspendisse non pellentesque sapien, et sagittis erat.
+            Quisque et dignissim elit. Sed sit amet sapien ac quam
+            vestibulum suscipit ac id diam. Morbi accumsan est vitae tellus
+            vulputate […]
+          </p>
+          <button class="btn btn-outline-secondary">READ MORE</button>
+        </div>
+      </div>
+    </div>
+    <a class="carousel-control-prev" href="#carouselExampleControls" role="button" data-slide="prev">
+      <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+      <span class="sr-only">Previous</span>
+    </a>
+    <a class="carousel-control-next" href="#carouselExampleControls" role="button" data-slide="next">
+      <span class="carousel-control-next-icon" aria-hidden="true"></span>
+      <span class="sr-only">Next</span>
+    </a>
+    </div>
+    <section class="container-subjects">
+      <div class="container">
+        <div class="row">
+          <div class="col-sm-4">
+            <h3>Web design</h3>
+            <p>Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vestibulum vehicula
+              erat non congue cursus.
+              Aenean auctor nulla quis augue dictum, sed sagittis odio varius.</p>
+          </div>
+          <div class="col-sm-4">
+            <h3>Web development</h3>
+            <p>Duis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed metus eget
+              mollis. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nspendisse at erat luctus.</p>
+          </div>
+          <div class="col-sm-4">
+            <h3>User interface</h3>
+            <p>Cras aliquet venenatis porttitor. Donec rutrum sapien et sapien dignissim, eu dapibus mi molestie.
+              Suspendisse dictum convallis quam et sodales. Praesent non enim et magna congue gravida.</p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="section-3">
```

Comment:
In this and the following portfolio sections there could be more space both on the right and left side.